services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/attribute/LocalDateTimeAttributeConverter.java

```diff
@@ -92,49 +95,59 @@ public static LocalDateTimeAttributeConverter create() {
 
     @Override
     public AttributeValueType attributeValueType() {
-        return AttributeValueType.N;
+        return AttributeValueType.S;
     }
 
     @Override
     public AttributeValue transformFrom(LocalDateTime input) {
         String value = "" +
-                       input.getYear() +
-                       padLeft2(input.getMonthValue()) +
-                       padLeft2(input.getDayOfMonth()) +
-                       padLeft2(input.getHour()) +
-                       padLeft2(input.getMinute()) +
+                       input.getYear() + "-" +
+                       padLeft2(input.getMonthValue()) + "-" +
+                       padLeft2(input.getDayOfMonth()) + "T" +
+                       padLeft2(input.getHour()) + ":" +
+                       padLeft2(input.getMinute()) + ":" +
                        padLeft2(input.getSecond()) +
                        (input.getNano() == 0 ? "" : "." + padLeft(9, input.getNano()));
-        return AttributeValue.builder().n(value).build();
+
+        return AttributeValue.builder().s(value).build();
     }
 
     @Override
     public LocalDateTime transformTo(AttributeValue input) {
-        if (input.n() != null) {
-            return EnhancedAttributeValue.fromNumber(input.n()).convert(VISITOR);
+        if (input.s() != null) {
+            return EnhancedAttributeValue.fromString(input.s()).convert(VISITOR);
         }
 
         return EnhancedAttributeValue.fromAttributeValue(input).convert(VISITOR);
     }
 
     private static final class Visitor extends TypeConvertingVisitor<LocalDateTime> {
         private Visitor() {
-            super(LocalDateTime.class, InstantAsIntegerAttributeConverter.class);
+            super(LocalDateTime.class, InstantAsStringAttributeConverter.class);
         }
 
         @Override
-        public LocalDateTime convertNumber(String value) {
-            String[] splitOnDecimal = ConverterUtils.splitNumberOnDecimal(value);
-            String[] chunkedDateTime = ConverterUtils.chunkWithLeftOverflow(splitOnDecimal[0], 2, 2, 2, 2, 2);
+        public LocalDateTime convertString(String value) {
+            // AttributeValue.S in LocalDateTime format
+            if (value.contains("T")) {
+                String[] splitOnDecimal = ConverterUtils.splitNumberOnDecimal(value);
+                String[] chunkedDateTime = ConverterUtils.chunkDateTime(splitOnDecimal[0]);
+
+                return LocalDateTime.of(Integer.parseInt(chunkedDateTime[0]),
+                                        Integer.parseInt(chunkedDateTime[1]),
+                                        Integer.parseInt(chunkedDateTime[2]),
+                                        Integer.parseInt(chunkedDateTime[3]),
+                                        Integer.parseInt(chunkedDateTime[4]),
+                                        Integer.parseInt(chunkedDateTime[5]),
+                                        Integer.parseInt(splitOnDecimal[1]));
```

Comment:
As discussed let's start with using parse and work from there. Let's do this for all the transformTo implementations